docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -44,12 +44,20 @@ Running dual-region setups requires the users to be able to detect any regional
 
 :::
 
+:::info
+
+The operational procedure was reworked in Camunda 8.6 to be more straightforward and reliable.
+
+The old procedure of 8.5 [as outlined](../../../../versioned_docs/version-8.5/self-managed/operational-guides/multi-region/dual-region-ops.md) is still compatible with Camunda 8.6 but compatibility will be removed in 8.7.
+
+:::
+
```

Comment:
```suggestion
:::info
This procedure has been updated in the Camunda 8.6 release. The procedure used in Camunda 8.5 has been deprecated, and compatibility will be removed in the 8.7 release.
:::
```
I'm so-so on if you'd like to include the link to the old version. If we're deprecating that method, I don't necessarily see the value in it, unless there are circumstances where the older procedure would be required. 

Can we move this up to the top of the page as well, for visibility?

Reply:
I'm fine not including the link, a person can just use the navbar on top to quickly switch the version.
It was just for people to have a grace period to readjust their own documentation.

Sure moved it to the top.